responder/jira/jira.go

```diff
@@ -0,0 +1,124 @@
+package jira
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/flanksource/commons/logger"
+	"github.com/flanksource/incident-commander/api"
+
+	jira "github.com/andygrunwald/go-jira"
+)
+
+type JiraOptions struct {
+	ProjectKey  string
+	Summary     string
+	Description string
+	IssueType   string
+}
+
+func GetClient(email, apiToken, url string) (*jira.Client, error) {
+
+	tp := jira.BasicAuthTransport{
+		Username: email,
+		Password: apiToken,
+	}
+
+	client, err := jira.NewClient(tp.Client(), url)
+	if err != nil {
+		return nil, err
+	}
+	return client, nil
+}
+
+func CreateIssue(client *jira.Client, opts JiraOptions) error {
```

Comment:
this should be on the client itself e.g.  func (client *jira.Client) CreateIssue(issue JiraIssue) (string, error)

Reply:
👍🏽